docs/featurestore/guides/featurestore.rst

```diff
@@ -268,45 +268,48 @@ Once the training dataset has been created, the dataset is discoverable in the f
     println(s"RMSE: ${trainingSummary.rootMeanSquaredError}")
     println(s"r2: ${trainingSummary.r2}")
 
-**Attaching Custom Metadata to a Feature Group**:
+Tagging Feature Groups and Training Datasets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The feature store APIs allows users to attach custom metadata to a feaure group. Currently, only Cached Feature Groups are supported. The users need to supply a metadata dictionary.
+The feature store enables users to attach tags to a featuregroup or training dataset in order to make them discoverable across feature stores.
+A tag is a simple {key: value} association, providing additional information about the data, such as for example geographic origin.
+This is useful in an organization so that data engineers can discover new featuregroups to work with and share them with other data engineers.
+Furthermore, making training datasets more easily discoverable for data scientists that they can work with, reducing duplicated work in terms of for example data preparation.
```

Comment:
Furthermore, making training datasets easier to discover for data scientists looking, reduces duplicated work in terms of for example data preparation.